spec/pallenec_spec.lua

```diff
@@ -22,21 +22,23 @@ describe("pallenec", function()
     end)
 
     it("Can compile pallene files", function()
-        util.shell("./pallenec test.pallene")
-        local out = util.shell("./lua/src/lua test_script.lua")
+        assert(util.execute("./pallenec test.pallene"))
+        local ok, err, out, _ = util.outputs_of_execute("./lua/src/lua test_script.lua")
```

Comment:
```suggestion
        local ok, err, out = util.outputs_of_execute("./lua/src/lua test_script.lua")
```

Reply:
I think the `_` makes it clearer that outputs_of_execute is returning 4 values and that we are intentionally ignoring the last one.